client/state/sites/constants.js

```diff
@@ -0,0 +1,61 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { join } from 'lodash';
+
+export const SITE_REQUEST_FIELDS = join( [
+	'ID',
```

Comment:
nice, so much more readable! Can we also sort them a-z so it's easier to spot dupes?

Reply:
All fields sorted and I also replaced Lodash with the native `Array.prototype.join`!